### PR TITLE
HBASE-24808 skip empty log cleaner delegate class names (WAS => cleaner.CleanerChore: Can NOT create CleanerDelegate= Cla…

### PR DESCRIPTION
**Pull request by saintstack**
…ssNotFoundException

### CODE REVIEW

File: hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java [hunk 1]
```diff
@@ -151,7 +151,11 @@ private void initCleanerChain(String confKey) {
     this.cleanersChain = new LinkedList<>();
     String[] logCleaners = conf.getStrings(confKey);
```
Comment (anoopsjohn) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java:152]: nit:  We can even rename this and related var names also.. This is not just logCleaner..  Hfile cleaner also.. may be call just cleaner or fileCleaner.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 36s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 40s |  branch-2 passed  |
| +1 :green_heart: |  checkstyle  |   1m  8s |  branch-2 passed  |
| +1 :green_heart: |  spotbugs  |   1m 59s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 15s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m  5s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  11m 37s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  spotbugs  |   2m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 15s |  The patch does not generate ASF License warnings.  |
|  |   |  33m  1s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2181/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2181 |
| Optional Tests | dupname asflicense spotbugs hadoopcheck hbaseanti checkstyle |
| uname | Linux 63d16f2ecdec 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 271a3cbf13 |
| Max. process+thread count | 94 (vs. ulimit of 12500) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2181/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by busbey** [comment]
Please change the jira subject and commit message to something like "skip empty log cleaner delegate class names" to describe what is changing

File: hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java (hunk 1 above)
Comment (saintstack) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java:152]: Can do in follow-on. Just reusing what was there ....

**Pull request merged**